libs/embed/package.json

```diff
@@ -94,6 +94,7 @@
     "@commitlint/cli": "^7.1.2",
     "@commitlint/config-conventional": "^7.1.2",
     "@rollup/plugin-replace": "^2.4.2",
+    "@types/estree": "^1.0.0",
```

Comment:
Was this one missing? not sure for what package it was related to

Reply:
you're right, I just remove it